README.md

```diff
@@ -14,12 +14,20 @@ pip install crunch-cli --upgrade
 
 ```python
 import crunch
-crunch = crunch.load_notebook(__name__)
+crunch = crunch.load_notebook()
 
 # Getting the data
 X_train, y_train, X_test = crunch.load_data()
 ```
 
+`crunch.load_data()` accept arguments for `read_parquet`.
```

Comment:
accepts, not accept. Also it's not clear what you mean. You want to say that the API is the same as pandas.read_parquet?

Reply:
Yes somehow, if you take a look at the example bellow, you will understand better.

I do not know how to word it better sorry.

Reply:
@matteoettam09 Right now in the CLI, `pandas.read_*` only uses default parameters. The aim is to accept all the parameters available for `pandas.read_csv` and `pandas.read_parquet`.

Reply:
Cool, fine